### ISSUE TITLE
Generated mdakit reStructuredText files should be put in a subdirectory

### ISSUE THREAD

**Issue by ianmkenney** [MDAnalysis/MDAKits#49]
The generated Kit documentation is placed in the same directory as manually edited docs, such as `about.rst`. For development this only creates noise when it comes to version control as each new kit will add a new line in the "Untracked files". I suggest we have generated files put into a `kits/` directory, which can then be put in the gitignore.

**Comment by IAlibay** [comment]
those files _should_ be autogenerated - they shouldn't ever be kept around

**Comment by IAlibay** [comment]
(oops accidentally clicked on "close with comment")

**Comment by ianmkenney** [comment]
Yes, when generating the files, the `kit/` directory should be created. No need to keep the empty directory around in VC either, basically the same way the `_build/` directory is treated which is explicitly in the gitignore.
